From: coolsnowwolf <coolsnowwolf@gmail.com>
Subject: fstools: avoid overlay zeroout operations

Disable discard and lazy initialization when creating the ext4 overlay.

Signed-off-by: coolsnowwolf <coolsnowwolf@gmail.com>

--- a/libfstools/common.c
+++ b/libfstools/common.c
@@ -222,10 +222,10 @@
 				snprintf(str, sizeof(str),
 					 "mkfs.f2fs -q -f -O extra_attr,compression -l rootfs_data %s", v->blk);
 			} else {
-				snprintf(str, sizeof(str), "mkfs.f2fs -q -f -l rootfs_data %s", v->blk);
+				snprintf(str, sizeof(str), "mkfs.f2fs -q -f -t 0 -l rootfs_data %s", v->blk);
 			}
 		} else {
-			snprintf(str, sizeof(str), "mkfs.ext4 -q -F -L rootfs_data %s", v->blk);
+			snprintf(str, sizeof(str), "UNIX_IO_NOZEROOUT=1 mkfs.ext4 -q -F -E nodiscard,lazy_itable_init=0,lazy_journal_init=0 -L rootfs_data %s", v->blk);
 		}
